.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        xcode:
+          - 11.5
+          - 11.6
+          - 12_beta
```

Comment:
@mmatoszko can you set Xcode `12` version only?

Reply:
done